onnxruntime/core/providers/cpu/rnn/deep_cpu_gru.cc

```diff
@@ -594,12 +594,6 @@ void UniDirectionalGru<T>::Compute(const gsl::span<const T>& inputs_arg,
   }
 
   {
-    // Enter a parallel section encompassing the kernels invoked
-    // below.  This lets the runtime system amortize loop entry/exit
-    // costs over a series of short kernels, and promotes cache
-    // affinity between iterations of successive loops.
-    onnxruntime::concurrency::ThreadPool::ParallelSection ps(ttp_);
```

Comment:
>ttp_ [](http://example.com/codeflow?start=61&length=4)

Why are we deleting this?
Does it have something to do with the stop/stop control?

Reply:
It has to do with the fact that any given thread can only have one ParallelSection at a time. Since we use one for bootstrap just prior to calling ExecuteGraph, there no need for this anymore.

Reply:
I thought the use of ParallelSection directly within InferenceSession::Run was just as an experiment.  Suggest we leave deep_cpu_gru.cc unchanged, and keep the spinning change in this PR separate.

Reply:
Will do so.